gtd-timer/Services/LogInService.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.IdentityModel.Tokens;
+using System;
+using System.Security.Claims;
+using System.Text;
+using System.IdentityModel.Tokens.Jwt;
+
+using gtdtimer.Timer.DAL.Entities;
+using gtdtimer.Timer.DTO;
+using gtdtimer.Constant;
+using gtdtimer.Timer.DAL.UnitOfWork;
+
+namespace gtdtimer.Services
+{
+    public class LogInService : ILogInService
+    {
+        private IUnitOfWork userManager;
+
+        public LogInService(IUnitOfWork userManager)
+        {
+            this.userManager = userManager;
+        }
+
+        public string CreateToken(LoginDTO model)
+        {
+            if (model == null)
```

Comment:
I believe our [ValidateModel] attr should catch this scenario (model being empty) - but please double check

Reply:
Yes, when user input email and password, this fields is checked
Okey, maybe delete this check

Reply:
with null reference